src/pages/experience/ExperienceCard.tsx

```diff
@@ -11,12 +11,27 @@ type ExperienceCardProp = {
   experience: Experience;
 };
 
+const parseShortMonthYear = (
+  dateStr: string
+): Date => {
```

Comment:
Love that you're specifying the return types for these functions. I should also be doing the same lol, this is what TypeScript is for!